src/pages/verify/integrations/javascript.mdx

```diff
@@ -0,0 +1,292 @@
+
+## Register Your Application in Criipto Verify
+
+After you signed up for Criipto Verify, you must [register an application](https://dashboard.criipto.com/applications/add) before you can try logging in with any e-ID.
+
+Once your application is registered, the dashboard will contain the necessary details to configure your application to communicate with Criipto Verify.
+You will need the following information from the General tab of your application settings:
+
+- _Client ID_ to identify your application to Criipto Verify.
+- _Domain_ on which you will be communicating with Criipto Verify.
+
+### Register callback URLs
+
+Before you can start sending authentication requests to Criipto Verify, you need to register the URLs on which you want to receive the returned *JSON Web Token*, JWT.
+
+The Callback URL of your application is the URL where Criipto Verify will redirect to after the user has authenticated in order for the OpenID Connect middleware to complete the authentication process.
+
+You will need to add this URL to the list of allowed URLs for your application.
+
+If you are developing with plain HTML and running your application locally, your callback URL might look like this:
+
+```
+http://localhost:3000/example/index.html
+```
+
+When you deploy your application to a different URL, you will also need to ensure you add that URL to the Callback URLs field in your application settings.
+
+## Install SDK
+
+You can include the library directly into your HTML file by loading it from [unpkg](https://www.npmjs.com/package/unpkg):
+
+```html
+<!-- index.html -->
+
+<script src="https://unpkg.com/@criipto/auth-js"></script>
+```
+
+Or install the package using [npm](https://npmjs.org/):
+
+```sh
+npm install --save @criipto/auth-js
+```
+
+## Initialize SDK with your application details
+
+If using the library from `unpkg`, you will have access to the global variable `CriiptoAuth`:
+
+```js
+// index.js
+
+var criiptoAuth = new CriiptoAuth({
+  domain: '{YOUR_CRIIPTO_DOMAIN}',
+  clientID: '{YOUR_CRIIPTO_APPLICATION_ID}',
+  store: sessionStorage,
+});
+```
+
+Or import the library if you are using npm:
+
+```js
+// index.js
+
+import CriiptoAuth from '@criipto/auth-js';
+
+var criiptoAuth = new CriiptoAuth({
+  domain: '{YOUR_CRIIPTO_DOMAIN}',
+  clientID: '{YOUR_CRIIPTO_APPLICATION_ID}',
+  store: sessionStorage,
+});
+```
+
+The `CriiptoAuth` constructor takes an object with the following parameters:
+
+- `domain (required, string)`: The domain your Criipto Application belongs to, such as `example.criipto.id`.
+- `clientID (required, string)`: The Client ID for your Criipto Application.
+
+Use your application's [domain and client ID](#register-your-application-in-criipto-verify) as the values for `domain` and `clientID` respectively.
+
+- `store (required)`: the library will store the JWT tokens in the specified location, `sessionStorage` or `localStorage`.
```

Comment:
This library doesn't actually store any JWT tokens, the store is used to maintain PKCE state when using redirect authentication.